app/src/main/java/ar/net/rainbyte/resistorvaluecalculator/MainActivity.kt

```diff
@@ -45,6 +71,18 @@ class MainActivity : Activity() {
 
         val multiplierValues: Array<String> = resources.getStringArray(R.array.multiplier_values)
         val toleranceValues: Array<String> = resources.getStringArray(R.array.tolerance_values)
+        val temperatureValues: Array<String> = resources.getStringArray(R.array.temperature_values)
+        val bandValues: Array<String> = resources.getStringArray(R.array.band_values)
+        resistorValueTemplate = resources.getString(R.string.resistor_value_template)
+        binding.spinnerTemperature.visibility = View.GONE
+        binding.spinnerFigure1.visibility = View.GONE
```

Comment:
these visibility default states could be defined in the activity_main.xml layout